### DISCUSSION TITLE
Transaction cancelled, please refer cancellation reasons for specific reasons [None, None, ValidationError]

### DISCUSSION THREAD

**Discussion by KennethWKZ** [dynamoose/dynamoose#1194] [Q&A] [answered]
How do i retrieve TransactionCanceledException errors from dynamoose.transaction?

**Comment by fishcharlie** [comment]
Not enough information to debug this.

**Reply by KennethWKZ** [reply]
No worries, I made a pull request for this. #1196

**Comment by KennethWKZ** [comment, marked as answer]
Here I made a pull request for this feature #1196
